CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#4892](https://github.com/bbatsov/rubocop/issues/4892): Add new `Lint/ShadowedArgument` cop. ([@akhramov][])
```

Comment:
I think that we should mention that this cop takes functionality away from `Lint/UnusedBlockArgument` and `Lint/UnusedMethodArgument`.

Reply:
Thanks, very good point, fixed.